script.js

```diff
@@ -58,15 +69,51 @@ class Bookshelf {
     if (title.value !== '' && author.value !== '') {
       const book = new Book(title.value, author.value);
       this.books.push(book);
-      this.savedata();
+      this.#savedata();
       bookShelf.innerHTML = '';
       this.retrievedata();
     }
   }
 }
```

Comment:
- [ ] (Optional) It would be nice to set the `title.value` and `author.value` to an empty string after adding a book. This would clear the input after you add a book. 👍